src/odemis/gui/cont/tabs.py

```diff
@@ -2919,65 +2926,18 @@ def __init__(self, name, button, panel, main_frame, main_data):
         panel.html_moi_doc.LoadPage(self.doc_path)
 
         if main_data.streak_cam:
```

Comment:
Let's also write `if main_data.streak_ccd`.
You should be able to entirely drop `.streak_cam` from the `MainGUIData` class.

Reply:
There are a couple of more places in the tab.py - also change?

Reply:
Yes :-)